app/models/rules/sealed_bid.rb

```diff
@@ -1,4 +1,4 @@
-class Rules::SealedBid < Rules::Basic
+class Rules::SealedBid < Rules::BaseRules
   def winning_bid
```

Comment:
been reading the Sandi Metz book and she had an example almost exactly like this so I followed her advice :)